src/hooks/useAppBootstrap.js

```diff
@@ -75,12 +76,23 @@ export const useAppBootstrap = client => {
         const payload = await Linking.getInitialURL()
         const { mainAppFallbackURL, cozyAppFallbackURL } =
           parseFallbackURL(payload)
+        if (mainAppFallbackURL || cozyAppFallbackURL) {
+          return setInitialRoute({
+            route: routes.home,
+            params: {
+              mainAppFallbackURL,
+              cozyAppFallbackURL
+            }
+          })
+        }
+
+        const defaultRedirectUrl = await getOrFetchDefaultRedirectionUrl(client)
 
         return setInitialRoute({
           route: routes.home,
           params: {
-            mainAppFallbackURL,
-            cozyAppFallbackURL
+            mainAppFallbackURL: undefined,
+            cozyAppFallbackURL: defaultRedirectUrl
```

Comment:
If `getOrFetchDefaultRedirectionUrl` returns `home` then we should set the `mainAppFallbackURL` instead.

Reply:
After reading the `getOrFetchDefaultRedirectionUrl()` method I understand that it would return `undefined` if `home` is default. So you implementation would work.

But someone said that the default url may contain some `path`. Is that still true? If yes it seems like the current implementation would not handle something like `home/#/connected/alan`

Reply:
Right!  I did not handle the case of a home with a path default_redirection. Fixed.